### ISSUE TITLE
Filtering with white space

### ISSUE THREAD

**Issue by mbnoimi** [axcore/tartube#543]
I want to use the following regex for passing `--reject-title` argument.
```regex
live|stream link|wide
```
But Tartube doesn't accept white space! instead it splits the expression into multiple arguments as following:

```
/home/laptop/.local/bin/yt-dlp --newline -i --match-filter !is_live --age-limit 21 -R 99 --hls-prefer-native --cookies /home/laptop/Downloads/youtube-dl/cookies.txt --write-sub --write-auto-sub --prefer-ffmpeg --no-cookies --write-desktop-link --sleep-requests 3 --sleep-subtitles 3 --reject-title live|stream --reject-title link|wide --output ...
```

I'm not pretty sure what if this is a bug or invalid used expression.

![image](https://user-images.githubusercontent.com/284547/235438769-e9025ecc-6117-4272-828c-45b7aef250fe.png)

**Comment by hiteshbedre** [comment]
Try: `--reject_title_list "link,wide"`
Reference: https://github.com/axcore/tartube/blob/ac738687337478015093a1550b0f994178538a06/tartube/options.py#L603

**Comment by mbnoimi** [comment]
> Try: `--reject_title_list "link,wide"` Reference:

When I pass 
```
--reject-title "live|stream link|wide"
```
It works fine. But using the GUI it doesn't!

**Comment by axcore** [comment]
What happens when you use `live|stream\slink|wide` ?

>Try: --reject_title_list "link,wide"

This will not work; `reject_title_list` is a name that Tartube uses internally.

**Comment by mbnoimi** [comment]
> What happens when you use `live|stream\slink|wide` ?

It works fine. Can you fix this issue in the GUI?

**Comment by mbnoimi** [comment]
> I'm away from my desk for a few weeks, but I will definitely investigate your problem when I get back.

Thanks

**Comment by axcore** [comment]
Should be fixed in v2.4.379.
